multi-node/aws/pkg/config/templates/cloud-config-controller

```diff
@@ -56,6 +76,29 @@ coreos:
         [Install]
         WantedBy=multi-user.target
 
+{{ if eq .ContainerRuntime "rkt" }}
+    - name: rkt-api.service
+      command: start
+      enable: true
+      content: |
+        [Service]
+        ExecStart=/usr/bin/rkt api-service
+        Restart=always
+        RestartSec=10
+
+    - name: load-rkt-stage1.service
+      command: start
+      content: |
+        [Unit]
+        Description=Load rkt stage1 images
+        Documentation=http://github.com/coreos/rkt
+        Requires=network-online.target
+        After=network-online.target
+        [Service]
+        Type=oneshot
+        ExecStart=/usr/bin/rkt fetch /usr/lib/rkt/stage1-images/stage1-coreos.aci /usr/lib/rkt/stage1-images/stage1-fly.aci  --insecure-options=image
```

Comment:
https://github.com/kubernetes/kubernetes/pull/27707

Doing this is currently necessary if we want rkt to cache the stage1 image making everything faster. I'm not sure why this is necessary instead of  it just caching after first use. 

@yifan-gu are there plans to make this unnecessary to run this unit for stage1 caching to work?